File: 2965-find-missing-and-repeated-values/2965-find-missing-and-repeated-values.java
class Solution {
    public int[] findMissingAndRepeatedValues(int[][] grid) {
        int n = grid.length;
        int N = n*n;
        int miss=-1;
        int rep = -1;
        int[] freq = new int[N+1];
        for(int i=0;i<n;i++){
            for(int j=0;j<n;j++){
                freq[grid[i][j]]++;
            }
        }
        for(int i=1;i<=n*n;i++){
            if(freq[i]==2){
                rep = i;
            }
            if(freq[i]==0){
                miss = i;
            }
        }
        return new int[]{rep,miss};
        
        
    }
}

File: 2965-find-missing-and-repeated-values/2965-find-missing-and-repeated-values.py
class Solution(object):
    def findMissingAndRepeatedValues(self, grid):
        n = len(grid)
        N = n*n;
        freq = [0]*(N+1);
        miss = -1;
        rep = -1;
        for i in range(n):
            for j in range(n):
                freq[grid[i][j]] +=1;
        for i in range(1, N+1):
            if(freq[i]==2):
                rep = i;
            if(freq[i]==0):
                miss=i;
        return [rep,miss];

        """
        :type grid: List[List[int]]
        :rtype: List[int]
        """
        